src/components/NcDateTimePickerNative/NcDateTimePickerNative.vue

```diff
@@ -50,6 +54,14 @@ All available types are: 'date', 'datetime-local', 'month', 'time' and 'week', p
 				label: 'please select a new date',
 			}
 		},
+		computed: {
+			valueString() {
+				if (this.value === null) {
```

Comment:
Maybe?
```suggestion
				if (typeof this.value === 'undefined' || this.value === null) {
```

Reply:
> Maybe?

or just

```js
if (this.value) {
```

as any Date is a truly value?

Or event just

```js
return this.value?.toString()
```

Reply:
Would also vote for
```js
return this.value?.toString() ?? 'null'
```

Reply:
Epic! I'll push the short variant.